test/remote/gateways/remote_checkout_v2_test.rb

```diff
@@ -439,7 +444,7 @@ def test_successful_purchase_with_ip
   end
 
   def test_failed_purchase
-    response = @gateway.purchase(12305, @credit_card, @options)
+    response = @gateway.purchase(100, @credit_card_dnh, @options)
     assert_failure response
     assert_equal 'Declined - Do Not Honour', response.message
```

Comment:
Checkout is responding with a `Invalid Card Number` for this error message 🤔

Reply:
Hi @aenand , I assume you are using legacy API creds that used our old Sandbox which relied on amounts (not card numbers) for simulations, and pointed to our previous gateway iteration which had a few differences in decline codes.

Do you have a processing channel at hand? (starting pc_…)

If not can you please create a new test account using this link please?
https://www.checkout.com/get-test-account

Let me know once the account is created and I’ll make sure it has everything configured for the remote tests to work

Thanks